apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/JourneyAppearance/Host/HostForm/HostForm.spec.tsx

```diff
@@ -204,4 +236,216 @@ describe('HostForm', () => {
     })
     expect(handleSelection).toHaveBeenCalledWith('selection')
   })
+
+  it('should call updatehost mutation on title change', async () => {
+    const updateHostMock: MockedResponse<UpdateHost, UpdateHostVariables> = {
+      request: {
+        query: UPDATE_HOST,
+        variables: {
+          id: 'hostId',
+          teamId: 'team.id',
+          input: {
+            title: 'Someone'
+          }
+        }
+      },
+      result: {
+        data: {
+          hostUpdate: {
+            id: 'hostId',
+            __typename: 'Host',
+            title: 'Title',
+            location: 'USA',
+            src1: 'imageSrc1',
+            src2: 'imageSrc2'
+          }
+        }
+      }
+    }
```

Comment:
```suggestion
      result: jest.fn(() => {
              data: {
          hostUpdate: {
            id: 'hostId',
            __typename: 'Host',
            title: 'Title',
            location: 'USA',
            src1: 'imageSrc1',
            src2: 'imageSrc2'
          }
        }
      })
```

This way you can get rid of line 266

Reply:
After chatting with Nisal, we're going to keep this as is